docs/pages/connect-your-client/gui-clients.mdx

```diff
@@ -296,6 +296,21 @@ more information.)
 ![DynamoDB local connection in NoSQL Workbench](../../img/database-access/nosql-workbench-connection.png)
 
 
+## SQL Server Azure Data Studio
+
+In Azure Data Studio create a connection using your proxy's endpoint. This is
+`localhost,62652` in the example above. Using the IP `127.0.0.1,62652` host could be required on
+a Windows machine instead of `localhost`. (See
+[Get connection information](./gui-clients.mdx#get-connection-information) for
+more information.)
+
+Create a connection with Microsoft SQL Server with the proxy endpoint `host`,`port`
```

Comment:
This paragraph is a bit jumbled to me. Would it be possible to use a bulleted list or table to indicate which values to use for which fields?